docs/index.html

```diff
@@ -0,0 +1,1176 @@
+<script src="heart_disease_predictor_report_files/libs/quarto-html/popper.min.js"></script>
+<script src="heart_disease_predictor_report_files/libs/quarto-html/tippy.umd.min.js"></script>
+<script src="heart_disease_predictor_report_files/libs/quarto-html/anchor.min.js"></script>
+<link href="heart_disease_predictor_report_files/libs/quarto-html/tippy.css" rel="stylesheet">
+<link href="heart_disease_predictor_report_files/libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
+<script src="heart_disease_predictor_report_files/libs/bootstrap/bootstrap.min.js"></script>
+<link href="heart_disease_predictor_report_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
+<link href="heart_disease_predictor_report_files/libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
+
+
+</head>
+
+<body>
+
+<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
+<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
+  <nav id="TOC" role="doc-toc" class="toc-active">
+    <h2 id="toc-title">Table of Contents</h2>
+   
+  <ul class="collapse">
+  <li><a href="#summary" id="toc-summary" class="nav-link active" data-scroll-target="#summary"><span class="header-section-number">1</span> Summary</a></li>
+  <li><a href="#introduction" id="toc-introduction" class="nav-link" data-scroll-target="#introduction"><span class="header-section-number">2</span> Introduction</a></li>
+  <li><a href="#methods" id="toc-methods" class="nav-link" data-scroll-target="#methods"><span class="header-section-number">3</span> Methods</a>
+  <ul class="collapse">
+  <li><a href="#data" id="toc-data" class="nav-link" data-scroll-target="#data"><span class="header-section-number">3.1</span> Data</a></li>
+  <li><a href="#preprocessing" id="toc-preprocessing" class="nav-link" data-scroll-target="#preprocessing"><span class="header-section-number">3.2</span> Preprocessing</a></li>
+  <li><a href="#analysis" id="toc-analysis" class="nav-link" data-scroll-target="#analysis"><span class="header-section-number">3.3</span> Analysis</a></li>
+  </ul></li>
+  <li><a href="#results-discussion" id="toc-results-discussion" class="nav-link" data-scroll-target="#results-discussion"><span class="header-section-number">4</span> Results &amp; Discussion</a>
+  <ul class="collapse">
+  <li><a href="#discussion" id="toc-discussion" class="nav-link" data-scroll-target="#discussion"><span class="header-section-number">4.1</span> Discussion</a></li>
+  </ul></li>
+  <li><a href="#references" id="toc-references" class="nav-link" data-scroll-target="#references"><span class="header-section-number">5</span> References</a></li>
+  </ul>
+<div class="quarto-alternate-formats"><h2>Other Formats</h2><ul><li><a href="../heart_disease_predictor_report.pdf"><i class="bi bi-file-pdf"></i>PDF</a></li></ul></div></nav>
```

Comment:
Is this something we can modify, or does it just get auto-generated by Quarto commands? If so, maybe someone can make a new issue noting that the PDF icon doesn't go anywhere valid